perllib/FixMyStreet/Cobrand/Bexley/Garden.pm

```diff
@@ -404,4 +409,12 @@ sub waste_get_paye_narrative {
     return "Garden Waste Service Payment - Reference: $id Contract: $uprn";
 }
 
+=item * Anyone can modify garden subs XXX Is that right?
```

Comment:
At the moment that's how WW works, but I imagine they may not want to go live with this until we've solved the problem with authenticating the person cancelling/amending the sub. (But not something to worry about in the PR.)

Reply:
Have removed the XXX bits for now, we'll see what Bexley come back with 👍